src/components/AboutMe.tsx

```diff
@@ -54,22 +145,30 @@ export default function AboutMe() {
                     <div className="mx-auto max-w-md px-4 sm:max-w-2xl sm:px-6 lg:px-0 lg:py-20 lg:max-w-none">
                         <blockquote>
                         <div>
-                            <svg
-                            className="h-12 w-12 text-light-brown"
-                            fill="currentColor"
-                            viewBox="0 0 32 32"
-                            aria-hidden="true"
-                            >
-                            <path d="M9.352 4C4.456 7.456 1 13.12 1 19.36c0 5.088 3.072 8.064 6.624 8.064 3.36 0 5.856-2.688 5.856-5.856 0-3.168-2.208-5.472-5.088-5.472-.576 0-1.344.096-1.536.192.48-3.264 3.552-7.104 6.624-9.024L9.352 4zm16.512 0c-4.8 3.456-8.256 9.12-8.256 15.36 0 5.088 3.072 8.064 6.624 8.064 3.264 0 5.856-2.688 5.856-5.856 0-3.168-2.304-5.472-5.184-5.472-.576 0-1.248.096-1.44.192.48-3.264 3.456-7.104 6.528-9.024L25.864 4z" />
-                            </svg>
                             <p className="mt-6 text-2xl font-medium text-white">
-                            Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed urna nulla vitae laoreet augue. Amet
-                            feugiat est integer dolor auctor adipiscing nunc urna, sit.
+                                I have experience in the following front-end tools:
```

Comment:
I think this description is too long, what do you think about something more simple like this:
```suggestion
                                Front-end technologies:
```

Reply:
![image](https://user-images.githubusercontent.com/58035837/156426166-7f0b0ada-ebc7-457c-b08b-899569ff3b42.png)
![image](https://user-images.githubusercontent.com/58035837/156426204-f825f068-6fce-4fa0-9b12-58a602e6847a.png)

I think it is better if it is long. Do not seems cool with the structure. The same happens with the switched container informations.

Reply:
okay